src/js/components/AppConfigEditFormComponent.jsx

```diff
@@ -160,6 +160,12 @@ var AppConfigEditFormComponent = React.createClass({
     });
   },
 
+  hasVIP: function () {
+    var features = this.props.features;
+
+    return Util.isArray(features) && features.includes("vips");
```

Comment:
You're right, I think we don't need that.
Question I can't answer myself.
Is the defaultProps is used if I pass `null` to the prop or only if I pass `undefined`.
If `null` is treated as a set prop it still makes sense to check for an array.